runtime/src/accounts_db.rs

```diff
@@ -90,6 +114,8 @@ pub struct AccountInfo {
 
     /// offset into the storage
     offset: usize,
+    /// size of the account
```

Comment:
well, is this comment really needed? Am I too smart to be able to guess from the field name?
instead how about this?:

``` rust
// needed to shrink by sparceness in bytes; the written-at-once accounts in a just-fit storage gradually overwrote by newer slots.
stored_size: usize,
```